Document IF NOT EXISTS, IF EXISTS, and CREATE OR REPLACE for secret DDL (#48272)
ydb-platform/ydb: ydb/docs/en/core/yql/reference/syntax/alter-secret.md

## Syntax {#syntax}

```sql
ALTER SECRET [IF EXISTS] secret_name
WITH (option = value[, ...])
```

> * `IF EXISTS` — do not output an error if the secret does not exist; in this case, the statement is a no-op.

Reviewer: ```suggestion
* `IF EXISTS` — the statement does not return an error if the secret does not exist; in this case, it is a no-op.
```
